src/components/SectionComponent.jsx

```diff
@@ -12,10 +12,14 @@ export default class SectionComponent extends Question {
   constructor(props) {
     super(props);
 
-    const toggleCollapseSuper = this._toggleCollapse;
-    this._toggleCollapse = (e) => {
+    const toggleCollapseSuper = this.toggleCollapse;
+    this.toggleCollapse = (e) => {
       let classList = e.target.classList;
-      if (!classList.contains('answer-content') && !classList.contains('card-header')) {
+      if (!classList.contains('answer-content') && !classList.contains('card-header')
+        && !classList.contains('answer') && !classList.contains('show-advanced-switch')
+        && !classList.contains('d-inline') && !classList.contains('caret-square-down')
+        && !(classList.length === 0)
```

Comment:
@LaChope This is very strange way of doing it :( but don't have idea how to solve it nicely